engine/src/flutter/third_party/canvaskit/BUILD.gn

```diff
@@ -70,6 +70,49 @@ copy("canvaskit_chromium_group") {
   ]
 }
 
+# This toolchain is only to be used by canvaskit_chromium_group below.
+wasm_toolchain("canvaskit_experimental_webparagraph") {
+  extra_toolchain_args = {
+    # In Chromium browsers, we can use the browser's APIs to get the necessary
+    # ICU data.
+    skia_use_icu = false
+    skia_use_client_icu = false
+    skia_use_bidi = true
+    skia_icu_bidi_third_party_dir = "//flutter/third_party/canvaskit/icu_bidi"
+
+    skia_enable_skparagraph = false
+    # TODO(mdebbar): This can't be set to false now because it causes font registration to fail.
+    # Once we update the font registration logic, we can come back and uncomment the next line.
+    # skia_canvaskit_enable_paragraph = false
+
+    # TODO(mdebbar): Set these to false once all image decoding can be done
+    # using the browser's built-in codecs.
+    # https://github.com/flutter/flutter/issues/122331
+
+    # In Chromium browsers, we can use the browser's built-in codecs.
```

Comment:
I know this is copypasta'd from above, but this comment is silly because it states the exact opposite of what we're actually doing with the flags below. In theory, we don't need these codecs, but in practice we do because there are still code paths that use the wasm codecs in canvaskit chromium. That's not really clear from the comment, the comment just says we don't need them, but then we include them anyway!

Reply:
Haha, yeah that comment is outdated. I'll remove it from here, and change the one above to a TODO.

Reply:
Done